k8s/rbac.yaml

```diff
@@ -26,6 +26,7 @@ metadata:
 subjects:
 - kind: ServiceAccount
   name: identityd
+  namespace: default
```

Comment:
It depends on which namespace user deployed, so I think it is better to let it empty or add an comment for it.

Reply:
Yes, I know that. 
This PR, I intended to be able to set up according to the README without changing the manifest.

Reply:
Yep, as @kevindiu mentioned, this is dependent on which namespace the user is deployed in so it was intended to be able to change based on what is needed.

Reply:
This is something we can configure through the helm template in https://github.com/yahoo/k8s-athenz-identity/pull/14

Reply:
I've updated that PR to make this service account field configurable with the namespace the identity service is deployed in.

Reply:
Thanks fo reply.
I agree that this will be configurable with helm template.